packages/validator/src/services/validatorStore.ts

```diff
@@ -168,14 +168,46 @@ export class ValidatorStore {
   }
 
   getFeeRecipient(pubkeyHex: PubkeyHex): Eth1Address {
-    return this.validators.get(pubkeyHex)?.feeRecipient ?? this.defaultProposerConfig.feeRecipient;
+    if (this.validators.has(pubkeyHex)) {
```

Comment:
on all these methods, it might be better to use this pattern:
```ts
const validatorData = this.validators.get(pubkeyHex);
if (validatorData === undefined) {
  throw Error(...);
}
return validatorData.x.y.z;
```